src/microbe_stage/player_hover_info.cpp

```diff
@@ -65,19 +65,31 @@ void
             "noCompounds", new Leviathan::BoolBlock(true)));
 
     } else {
-        for(const auto& tuple : compounds) {
 
-            std::stringstream compoundInfo;
-            compoundInfo << std::fixed << std::setprecision(2)
-                         << SimulationParameters::compoundRegistry
-                                .getTypeData(std::get<0>(tuple))
-                                .displayName
-                         << ": " << std::get<1>(tuple);
+        Json::Value compoundsJson;
```

Comment:
This could be `Json::Value compoundsJson(Json::arrayValue);` to make sure that this is always an array even when empty.